### PR TITLE
Scrum 73 74

### PR DESCRIPTION
**Pull request by connerk2001**
Bugs involving attendances showing all entries and not being a foreign key to users fixed. Attendances will now cascade upon deletion of a user. Also fixed some small issues with styling such as the top bar color difference and flash notices not appearing on attendances and events pages.

### CODE REVIEW

File: app/assets/stylesheets/application.css
Comment (DineshBa25) [inline app/assets/stylesheets/application.css:1]: I dont think you meant to remove this from the whole app css, this is gonna make the app bar look ugly

**Review by DineshBa25 — COMMENTED** [review]
Everything else looks good and matches the requirements for SCRUM-73. But I dont see where anything is being done here for SCRUM-74. Did you intend to include that number in the tile of this pull request?

File: app/assets/stylesheets/application.css
Comment (connerk2001) [reply app/assets/stylesheets/application.css:1]: Yes I believe that the changes I made only remove the top-bar class which is an old class that was originally used to style the CRUD pages in sprint 1. It is not related at all to the app bar.

Comment (connerk2001) [reply app/assets/stylesheets/application.css:1]: The changes related to SCRUM-74 are those in the model files. All that was needed is the dependent: :destroy in order to cascade upon the deletion of a user.

Comment (DineshBa25) [reply app/assets/stylesheets/application.css:1]: > Yes I believe that the changes I made only remove the top-bar class which is an old class that was originally used to style the CRUD pages in sprint 1. It is not related at all to the app bar.

You are correct, i mistakenly believed this was referring to the navbar.



> The changes related to SCRUM-74 are those in the model files. All that was needed is the dependent: :destroy in order to cascade upon the deletion of a user.

Sounds good, just making sure.

**Review by DineshBa25 — APPROVED** [review]
Everything looks great as far as fulfilling the requirements of the Jira and the code seems to match our teams coding standard.

**Pull request merged**